src/index.html

```diff
@@ -17,41 +17,41 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="#">
         <img src="./images/logo.png" alt="Moyo logo">
       </a>
 
-      <nav class="nav">
+      <nav class="nav nav--">
         <ul class="nav__list">
-          <li class="nav__item nav__item--is-active">
-            <a href="" class="nav__link">Apple</a>
+          <li class="nav__item nav__item--position nav__item--is-active">
+            <a href="#" class="nav__link">Apple</a>
           </li>
-          <li class="nav__item">
-            <a href="" class="nav__link">Samsung</a>
+          <li class="nav__item nav__item--position">
+            <a href="#" class="nav__link">Samsung</a>
           </li>
-          <li class="nav__item">
-            <a href="" class="nav__link">Smartphones</a>
+          <li class="nav__item nav__item--position">
+            <a href="#" class="nav__link">Smartphones</a>
           </li>
-          <li class="nav__item">
-            <a href="" class="nav__link">Laptops & computers</a>
+          <li class="nav__item nav__item--position">
+            <a href="#" class="nav__link">Laptops & computers</a>
           </li>
-          <li class="nav__item">
-            <a href="" class="nav__link">Gadgets</a>
+          <li class="nav__item nav__item--position">
+            <a href="#" class="nav__link">Gadgets</a>
           </li>
-          <li class="nav__item">
-            <a href="" class="nav__link">Tablets</a>
+          <li class="nav__item nav__item--position">
+            <a href="#" class="nav__link">Tablets</a>
           </li>
-          <li class="nav__item">
-            <a href="" class="nav__link">Photo</a>
+          <li class="nav__item nav__item--position">
+            <a href="#" class="nav__link">Photo</a>
           </li>
-          <li class="nav__item">
-            <a href="" class="nav__link">Video</a>
+          <li class="nav__item nav__item--position">
```

Comment:
wrong usage of modifier
https://en.bem.info/methodology/quick-start/#modifier

Reply:
changed them too 👍